.ci/install_podman.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2029 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+default_kata_config="/usr/share/defaults/kata-containers/configuration.toml"
+kata_config="/etc/kata-containers/configuration.toml"
+
+setup_prerequisites() {
+	# Verify host kernel version
+	host_kernel_version=$(uname -r|cut -d. -f1-2)
+	result=$(echo "${host_kernel_version} >= 4.14"|bc)
+	[ "${result}" -ne 1 ] && echo "ERROR: host kernel version too old" && exit 1
```

Comment:
Nit: `echo >&2 "ERROR: ..."`.

Reply:
we could also include our `lib.sh` and use our standard `die` func...